packages/core/src/routes/experience/classes/experience-interaction.ts

```diff
@@ -65,7 +68,12 @@ const interactionStorageGuard = z.object({
   profile: interactionProfileGuard.optional(),
   mfa: mfaDataGuard.optional(),
   verificationRecords: verificationRecordDataGuard.array().optional(),
-  oneTimeTokenContext: oneTimeTokenContextGuard.optional(),
+  oneTimeToken: z
+    .object({
+      verified: z.boolean(),
+      context: oneTimeTokenContextGuard.optional(),
+    })
```

Comment:
It should be placed within the verification record.

Reply:
Updated